src/gfx/CMakeLists.txt

```diff
@@ -40,6 +40,9 @@ target_precompile_headers(gfx PUBLIC "$<$<COMPILE_LANGUAGE:CXX>:${CMAKE_CURRENT_
 add_library(gfx_texture_file texture_file/texture_file.cpp)
 target_link_libraries(gfx_texture_file gfx)
 
+add_library(gfx_gltf gltf/gltf.cpp)
+target_link_libraries(gfx_gltf gfx tinygltf)
```

Comment:
Good practice :) I'm also learning modern CMake recently. You probably already know this, but I learned a few tips from this guy: https://www.youtube.com/watch?v=bsXLMQ6WgIk